### PR TITLE
DEDP proctored exam grades

### PR DESCRIPTION
**Pull request by rachellougee**
### What are the relevant tickets?

https://github.com/mitodl/hq/issues/4481

### Description (What does it do?)

- Creating int__mitxonline__proctored_exam_grades
- Updating marts__micromasters_dedp_exam_grades to add the missing grades from MITx Online

### How can this be tested?

dbt build --select int__mitxonline__proctored_exam_grades
dbt build --select marts__micromasters_dedp_exam_grades

### CODE REVIEW

**Review by KatelynGit — APPROVED** [review]
The builds looked good, table testing went well, and definitions looked good. Based on peter's feedback we should open an issue to look into possibly adding data to proctoredexamgrade_passing_score in the future though.

**Comment by rachellougee** [comment]
I made changes to add proctoredexamgrade_passing_grade in the same format as proctoredexamgrade_percentage_grade in the mart. Also, the semester is no longer null in the mart as I populated the old runs in MicroMasters yesterday. 
I re-ran the dbt code to ensure the changes are working as expected.

File: src/ol_dbt/models/intermediate/mitxonline/_int_mitxonline__models.yml [hunk 1]
```diff
@@ -1905,3 +1905,61 @@ models:
...
+  description: exam grades either from MITx Online exam course runs or the proctored
+    exam unit in MITx Online courses
+  columns:
+  - name: course_number
+    description: str, unique string for the course. It can contain letters, numbers,
+      or periods. e.g. 14.310x
+    tests:
+    - not_null
+  - name: course_title
+    description: str, title of the course that the exam grade associated with
```
Comment (pdpinch) [inline src/ol_dbt/models/intermediate/mitxonline/_int_mitxonline__models.yml:1919]:
```suggestion
    description: str, title of the course associated with the exam grade
```

File: src/ol_dbt/models/marts/micromasters/_marts_micromasters__models.yml [hunk 2]
```diff
@@ -3,57 +3,48 @@ version: 2
...
-    description: str, title of the program
-    tests:
-    - not_null
-  - name: micromasters_program_id
-    description: int, id of the program in the micromasters database
-  - name: mitxonline_program_id
-    description: int, id of the program in the mitxonline database
   - name: course_title
-    description: str, title of the course
+    description: str, title of the course that the exam grade associated with
```
Comment (pdpinch) [inline src/ol_dbt/models/marts/micromasters/_marts_micromasters__models.yml:9]:
```suggestion
    description: str, title of the course associated with the exam grade
```

File: src/ol_dbt/models/intermediate/mitxonline/_int_mitxonline__models.yml [hunk 3]
```diff
@@ -1905,3 +1905,61 @@ models:
...
+  - name: user_email
+    description: str, user email on MITx Online
+    tests:
+    - not_null
+  - name: proctoredexamgrade_passing_grade
+    description: float, passing grade for the exam run, range between 0 to 1. Null
+      for MITx Online courses after 2T2022.
+  - name: proctoredexamgrade_grade
+    description: float, user's grade for the proctored exam, range between 0 to 1
+      with 4 decimals.
```
Comment (pdpinch) [inline src/ol_dbt/models/intermediate/mitxonline/_int_mitxonline__models.yml:1956]: for consistency? 
```suggestion
    description: float, user's grade for the proctored exam, range between 0 to 1
      with 4 decimals,  e.g., 0.8251
```

**Review by pdpinch — APPROVED** [review]
Just a couple of small tweaks to field descriptions. Take them or leave them.

**Comment by rachellougee** [comment]
Thank you, Kate and Peter, for the review.

**Pull request merged**